packages/fields/src/types/CloudinaryImage/ImageBlock.js

```diff
@@ -4,12 +4,25 @@ import CloudinaryImage from './';
 import SelectType from '../Select';
 import RelationshipType from '../Relationship';
 
+const RelationshipWrapper = {
+  ...RelationshipType,
+  implementation: class extends RelationshipType.implementation {
+    async resolveNestedOperations(operations, item, context, ...args) {
+      const result = await super.resolveNestedOperations(operations, item, context, ...args);
```

Comment:
Taking advantage of work from https://github.com/keystonejs/keystone-5/pull/1033